sdk/documenttranslation/Azure.AI.DocumentTranslation/README.md

```diff
@@ -1,19 +1,488 @@
 # Azure Cognitive Services Document Translation client library for .NET
+Document Translation is a cloud-based feature of the Azure Translator service and is part of the Azure Cognitive Service family of REST APIs. The Document Translation API translates documents to and from 90 languages and dialects while preserving document structure and data format.
+
+[Source code][documenttranslation_client_src] | [Package (NuGet)][documenttranslation_nuget_package] | [API reference documentation][documenttranslation_refdocs] | [Product documentation][documenttranslation_docs] | [Samples][documenttranslation_samples]
 
 ## Getting started
 
 ### Install the package
+Install the Azure Document Translation client library for .NET with [NuGet][nuget]:
+
+```PowerShell
+dotnet add package Azure.AI.DocumentTranslation
```

Comment:
because we haven't GAd this package, we need to add `--prerelease` to the end of this line